docs/managing-dragonfly/replication.md

```diff
@@ -39,3 +39,6 @@ replicaof no one
 
 This will stop the instance from replicating the primary server but will not discard the dataset it has already replicated. This syntax is useful in cases where the original primary fails. After running `replicaof no one` on a replica of the failed primary, the former replica can be used as the new primary and have its own replicas as a failsafe.
 
+## Monitoring Lag
+
+Dragonfly defines the replication lag as the maximum amount of unacknowledged database changes over all the shards. This metric is calculated by the master instance and is visible both as the `dfly_connected_replica_lag_records` field in the [prometheus metrics](./monitoring.md), and through the `INFO REPLICATION` command.
```

Comment:
what do you mean by maximum?
When I read this, it sounds like some kind of limit and not a counter

Reply:
does 'maximal' sound better? I mean 'maximum' here as opposed to 'the total amount/sum'. Like if the lags in the shards are `5,6,20` then the lag is `20`, not `31`.

Reply:
I see, So maybe write maximal among all shard